src/net.h

```diff
@@ -969,6 +969,12 @@ class CConnman
     void RecordBytesRecv(uint64_t bytes);
     void RecordBytesSent(uint64_t bytes) EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex);
 
+    /**
+     Return networks for which we have no addresses in addrman and therefore
+     may require loading fixed seeds.
+     */
+    std::vector<Network> GetRelevantFixedSeedNetworks() const;
```

Comment:
naming: "fixed seeds" seems odd to put in the function name because that's the caller - doesn't really have to do with the logic of the function. I think the functionality would better be captured with something like `GetReachableEmptyNetworks()`

Reply:
renamed, thanks